lib/pacemaker/pcmk_sched_notif.c

```diff
@@ -782,35 +782,39 @@ find_remote_start(pe_action_t *action)
     return NULL;
 }
 
-void
-create_notifications(pe_resource_t * rsc, notify_data_t * n_data, pe_working_set_t * data_set)
+/*!
+ * \internal
+ * \brief Create notify actions, and add notify data to original actions
+ *
+ * \param[in] rsc       Clone or clone instance that notification is for
+ * \param[in] n_data    Clone notification data for some action
+ * \param[in] data_set  Cluster working set
+ */
+static void
+create_notify_actions(pe_resource_t *rsc, notify_data_t *n_data,
+                      pe_working_set_t *data_set)
```

Comment:
Could rsc->cluster be used here as well?  I've not been keeping an eye out for this everywhere, but it seems like there's probably a lot of functions that could have this change made.

Reply:
Yes on both counts. At some point I started doing that but I haven't gone back to functions I already messed with.